devicetypes/smartthings/lifx-color-bulb.src/lifx-color-bulb.groovy

```diff
@@ -145,15 +145,23 @@ def setLevel(percentage) {
 	log.debug "setLevel ${percentage}"
 	if (percentage < 1 && percentage > 0) {
 		percentage = 1 // clamp to 1%
-	}
+	} else {
+    	try {
+        	percentage = Math.round(percentage)
+        } catch (Exception ex) {
+            log.Error "Caught exception while converting value '$percentage' to integer: $ex"
```

Comment:
should be "log.error", not capital "E". Not sure if catching exception is even necessary.